File: src/writingcat/entity/CollocationDetail.ts
import { Interpretation } from "./Interpretation";

export default class CollocationDetail {
    static issue: [string];
    static collocation: string;
    static note: string;
    static wordKey: string;
    static interpretation: [Interpretation];
    public constructor(issue: [string], collocation: string, note: string, wordKey: string, interpretation: [Interpretation]) {
        issue = issue;
        collocation = collocation;
        note = note;
        wordKey = wordKey;
        interpretation = interpretation;
    }

    public static issueStr(): string {
        return "issues";
    }
    public static collocationStr(): string {
        return "collocation";
    }
    public static noteStr(): string {
        return "note";
    }
    /**
     * wordKeyName
     */
    public static wordKeyStr(): string {
        return "wordKeys";
    }
    public static interpretationStr(): string {
        return "interpretation";
    }
}

File: Home_WritingCat/src/main/java/writingCat/entity/CollocationDetail.java
package writingCat.entity;

import lombok.Builder;

/**
 * @Description: CollocationDetail
 * @Author: wx:istarwyh
 * @Date: 2021-03-19 02:34
 * @Version: ing
 */
@Builder
public class CollocationDetail {
    public String[] issues;
    public String collocation;
    public String[] wordKeys;
    public String note;
    public Interpretation[] interpretations;
}
